core/arch/arm/plat-nuvoton/conf.mk

```diff
@@ -0,0 +1,54 @@
+PLATFORM_FLAVOR ?= npcm845x
+
+ifeq ($(PLATFORM_FLAVOR),npcm845x)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+CFG_ARM64_core = y
+endif #npcm845x
+CFG_USER_TA_TARGETS ?= ta_arm64
+DEBUG ?= 1
+
+ifeq ($(platform-debugger-arm),1)
+# ARM debugger needs this
+platform-cflags-debug-info = -gdwarf-2
+platform-aflags-debug-info = -gdwarf-2
+endif
+
+ifeq ($(platform-flavor-armv8),1)
+CFG_WITH_ARM_TRUSTED_FW = y
+endif
+
+CFG_GIC = y
+CFG_ARM_GICV2 = y
+CFG_SECURE_TIME_SOURCE_CNTPCT = y
+CFG_16550_UART = y
+
+ifeq ($(CFG_CORE_TPM_EVENT_LOG),y)
+# NOTE: Below values for the TPM event log are implementation
+# dependent and used mostly for debugging purposes.
+# Care must be taken to properly configure them if used.
+# CFG_TPM_LOG_BASE_ADDR ?= 0x402c951
+# CFG_TPM_MAX_LOG_SIZE ?= 0x200
+endif
+
+ifeq ($(CFG_ARM64_core),y)
+CFG_WITH_LPAE = y
```

Comment:
not needed. Alread addressed in _core/arch/arm/arm.mk_.

Reply:
Done